.codeclimate.yml

```diff
@@ -1,6 +1,10 @@
 engines:
   fixme:
-    enabled: false
+    enabled: true
+    config:
+      strings:
+      - legacy
+      - WIP
```

Comment:
My (weakly held) opinion is that I think it's less confusing to use the README for education and leave it disabled here.